src/main/java/com/amazon/kinesis/streaming/agent/metrics/Metrics.java

```diff
@@ -34,8 +34,9 @@
     public static final Pattern PARSER_TOTAL_RECORDS_PROCESSED_METRIC = Pattern.compile("^.*Parser.TotalRecordsProcessed$");
     public static final Pattern PARSER_TOTAL_RECORDS_SKIPPED_METRIC = Pattern.compile("^.*Parser.TotalRecordsSkipped$");
 
+    public static final String CLASS_DIMENSION = "Class";
     public static final String DESTINATION_DIMENSION = "Destination";
-    public static final String INSTANCE_DIMENSION = "Instance";
+    public static final String INSTANCE_DIMENSION = "InstanceId";
```

Comment:
I want to note here that the Amazon EC2 metrics all use the `InstanceId` dimension. In Datadog, we want (if possible) for our custom CloudWatch metrics to nicely tie directly to an EC2 Instance and all of its tags/etc.. so I am changing this. I need to talk to @Chaochenq about this though too.